src/openrct2/ride/TrackPaint.h

```diff
@@ -28,6 +30,21 @@ extern const uint8_t edges_4x4[];
 
 extern const uint8_t track_map_1x4[][4];
 
+namespace TrackPaint::Flat
+{
+    // Lift and No lift sprites
+    using BasicSprites = std::array<std::array<uint16_t, NumOrthogonalDirections>, 2>;
+
+    // No lift sprites
+    using NoLiftSprites = std::array<uint16_t, NumOrthogonalDirections>;
+
+    // Lift and No Lift (sprites * 2 {front, back})
+    using ComplexSprites = std::array<std::array<std::array<uint16_t, 2>, NumOrthogonalDirections>, 2>;
```

Comment:
Can you give an example of this?

Reply:
Bobsleigh or do you mean in my comment give an example

Reply:
You mean that every piece consists of two sprites, or?

Reply:
yes